### ISSUE TITLE
Issue with seconds and  milliseconds resolutions 

### ISSUE THREAD

**Issue by webInfrastructure** [tradingview/charting-library-tutorial#71]
I have a problem with resolutions . 
Firstly, The chart doesn't show seconds revolutions, but I've already added seconds in supported_resolutions
Secondly, I need to use : 0,5S, 0.25S resolutions, Unfortunately, I could not find any information about values less then 1 second 
my configurationData:
`const configurationData = {
  supported_resolutions: [
    '3S',
    '5S',
    '10S',
    '15S',
    '30S',
    '1',
    '3',
    '5',
    '15',
    '30',
    '1h',
    '2h',
    '3h',
    '4h',
    '8h',
    '1D',
    '1W',
    '2W',
    '1M',
  ],
  exchanges: [
    {
      value: 'BINANCE',
      name: 'BINANCE',
      desc: 'BINANCE',
    },
  ],
  symbols_types: [
    {
      name: 'crypto',

      // `symbolType` argument for the `searchSymbols` method, if a user selects this symbol type
      value: 'crypto',
    },
    // ...
  ],
}`

and my symbolInfo:
`const symbolInfo = {
      ticker: symbolItem.full_name,
      name: symbolItem.symbol,
      description: symbolItem.description,
      type: symbolItem.type,
      session: '24x7',
      timezone: 'Etc/UTC',
      exchange: symbolItem.exchange,
      minmov: 1,
      pricescale: 100,
      minmove2: 0,
      has_seconds: true,
      has_intraday: true,
      has_no_volume: false,
      has_empty_bars: true,
      has_weekly_and_monthly: true,
      supported_resolutions: configurationData.supported_resolutions,
      volume_precision: 3,
      data_status: 'streaming',
      options: symbolItem.options,
      pair_id: symbolItem.pair_id,
    }`

**Comment by tarikhagustia** [comment]
hi, i have same issue. can you set resolutions in 1 minutes? cause i can't

**Comment by webInfrastructure** [comment]
@tarikhagustia Yes, 1 minute and other (bigger then 1M) work well. Do you use widget or chart ?

**Comment by SlicedSilver** [comment]
To enable second resolutions you need to enable the `seconds_resolution` featureset (it is disabled by default). https://github.com/tradingview/charting_library/wiki/Featuresets#important-features

Additionally, ensure that your SymbolInfo has the correct values for:
- https://github.com/tradingview/charting_library/wiki/Symbology#has_seconds
- https://github.com/tradingview/charting_library/wiki/Symbology#seconds_multipliers

For sub second resolutions you will probably also be interested in:
- https://github.com/tradingview/charting_library/wiki/Symbology#has_ticks
- `tick_resolution` featureset

Additionally, I would advise posting questions like this on the [charting_library repo](https://github.com/tradingview/charting_library/issues) instead.

**Comment by tarikhagustia** [comment]
thank you, solved!

**Comment by Novenqingtang** [comment]
Hello, how did you solve this problem? @tarikhagustia

**Comment by Novenqingtang** [comment]
> To enable second resolutions you need to enable the `seconds_resolution` featureset (it is disabled by default). https://github.com/tradingview/charting_library/wiki/Featuresets#important-features
> 
> Additionally, ensure that your SymbolInfo has the correct values for:
> 
> * https://github.com/tradingview/charting_library/wiki/Symbology#has_seconds
> * https://github.com/tradingview/charting_library/wiki/Symbology#seconds_multipliers
> 
> For sub second resolutions you will probably also be interested in:
> 
> * https://github.com/tradingview/charting_library/wiki/Symbology#has_ticks
> * `tick_resolution` featureset
> 
> Additionally, I would advise posting questions like this on the [charting_library repo](https://github.com/tradingview/charting_library/issues) instead.

Hello, i have the same problem, and these links is 404.

**Comment by Novenqingtang** [comment]
<img width="656" alt="image" src="https://github.com/tradingview/charting-library-tutorial/assets/20578889/d4857669-f0cf-44a1-9ec3-7b51406a88ae">
This is my demo code, and the seconds_multipliers field is not effect.
